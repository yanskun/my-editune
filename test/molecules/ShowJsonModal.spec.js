import { createLocalVue, shallowMount } from "@vue/test-utils"
import ElementUI from "element-ui"
import Component from "~/components/molecules/ShowJsonModal"

const localVue = createLocalVue()
localVue.use(ElementUI)

describe("ShowJsonModal", () => {
  test("snapshot", () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        abilities: [],
        open: true
      },
      localVue
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
