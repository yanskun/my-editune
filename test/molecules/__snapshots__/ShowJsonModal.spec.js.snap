// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowJsonModal snapshot 1`] = `
<el-dialog-stub
  closeonclickmodal="true"
  closeonpressescape="true"
  customclass=""
  lockscroll="true"
  modal="true"
  modalappendtobody="true"
  modalfade="true"
  showclose="true"
  title=""
  top="15vh"
  visible="true"
>
  <span>
    <el-button-stub
      icon=""
      nativetype="button"
      type="default"
    >
      閉じる
    </el-button-stub>
    <el-button-stub
      icon=""
      nativetype="button"
      type="default"
    >
      コピーする
    </el-button-stub>
  </span>
</el-dialog-stub>
`;
